Sources/Confidence/EventStorage.swift

```diff
@@ -0,0 +1,88 @@
+import Foundation
+
+internal protocol EventStorage {
+    func startNewBatch() throws
+    func writeEvent(event: Event) throws
+    func batchReadyFiles() throws -> [URL]
+    func eventsFrom(fileURL: URL) throws -> [Event]
```

Comment:
we need to also add a remove function to delete the file at url to cleanup the successful update.